### DISCUSSION TITLE
Support version 1.1.0 of CIS Benchmarks for Amazon EKS

### DISCUSSION THREAD

**Discussion by trallnag** [aquasecurity/kube-bench#1202] [Development]
A new version of the benchmark is available.

![image](https://user-images.githubusercontent.com/24834206/172615231-ab571648-5963-4bb2-8861-a35bb982a56a.png)

Support for this would be great. Currently the latest version of `kube-bench` does not support it.

```txt
sh-4.2$ kube-bench run --targets node --benchmark eks-1.1.0
error validating targets: No targets configured for eks-1.1.0
```

**Comment by mozillazg** [comment]
@trallnag Thanks. We will take a look into it.
